server/bin/pbench-server-put-shim-001

```diff
@@ -0,0 +1 @@
+pbench-trampoline
```

Comment:
Will the trampoline be able to bounce the user from `pbench-server-put-shim-001` (with hyphens) to `pbench_server_put_shim_001.py` (with underscores)?

Also, is there really a requirement/demand for this use of the trampoline?  That is, this is basically new code for a one-time, hand-produced deployment...does it really need lots of command-line support?

Reply:
Indeed; none of the "server commands" are intended to be activated interactively; they're only triggered by the `cron` jobs. The trampoline just looks for (basename).py and therefore won't be able to perform this translation. (Though we could add that name filter to the trampoline if we really wanted to, it seems pointless in this context.)

As a general architectural goal, we should be pushing as much of the server behavior as possible into Python modules, outside the pseudo-CLI level, because my goal is to drop all of the command invocations entirely and move to a pure message-passing microcomponent model. Which means any actual behavior in the "CLI" modules will just need to be moved anyway.